examples/style-expressions.js

```diff
@@ -28,6 +28,7 @@ const map = new Map({
         'circle-fill-color': 'gray',
         'circle-stroke-color': 'white',
         'circle-stroke-width': 0.5,
+        'text-value': ['coalesce', ['get', 'missing'], 'default value'],
```

Comment:
I assumed this was accidentally left in and removed it since it made the example quite busy.

<img width="941" alt="image" src="https://github.com/openlayers/openlayers/assets/41094/5619f9e2-82eb-4d96-b1f7-7ab20f222658">